modules/cxx.rs/1.0.145/presubmit.yml

```diff
@@ -0,0 +1,15 @@
+matrix:
+  platform:
+    - macos_arm64
+    - ubuntu2404
+    - windows
+  bazel: [7.x, 8.x]
```

Comment:
```suggestion
  bazel: [7.x, 8.x, rolling]
```

Reply:
Why? Isn't this risky where unreleased changes lead to build failures?